ansible_collections/arista/avd/roles/eos_designs/templates/inband_management/parent-logic.j2

```diff
@@ -1,15 +1,18 @@
-{# Add underlay links defined on this device #}
 {% set inband_management_data.vlans = [] %}
 {% set inband_management_data.subnets = [] %}
 
-{# Add underlay links defined on other devices pointing to this device #}
-{% for fabric_switch in groups[fabric_name] | arista.avd.natural_sort %}
-{%     set fabric_switch_facts = hostvars[fabric_switch].switch | arista.avd.default %}
-{%     if inventory_hostname in fabric_switch_facts.inband_management_parents | arista.avd.default([]) %}
-{%         set inband_management_data.role = 'parent' %}
-{%         if fabric_switch_facts.inband_management_subnet not in inband_management_data.subnets %}
-{%             do inband_management_data.vlans.append(fabric_switch_facts.inband_management_vlan) %}
-{%             do inband_management_data.subnets.append(fabric_switch_facts.inband_management_subnet) %}
+{# Add inband management variables defined on other devices pointing to this device #}
+{% set avd_topology_peers = hostvars[ansible_play_hosts[0]].ansible_facts.avd_topology_peers[inventory_hostname] | arista.avd.default([]) %}
```

Comment:
Do we need to add a defined check for 'hostvars' . Like ``` if hostvars is arista.avd.defined ```

Reply:
hostvars is an "Ansible magic variable", so it is always available.